src/handlers/events/me.ts

```diff
@@ -0,0 +1,299 @@
+import * as _ from 'lodash';
+import * as db from '../../database';
+import { formatDate } from '../../utils';
+import type { Context } from '../../types/telegraf';
+import logger from '../../logger';
+import * as messages from '../../constants/messages';
+import { sendExpl } from '../../utils';
+import { User } from 'telegraf/typings/telegram-types';
+
+const DATA_SEPARATOR = '|';
+const EXPLS_IN_PAGE = 5;
+
+function getListTitle(page: string) {
+  switch (page) {
```

Comment:
instead of switch-case, map would be a cleaner solution
```suggestion
  const messageMap = {
    latest: messages.me.latestByYou(),
    // ...
  };
  return messageMap[page] || '';
```